{{cookiecutter.github_project_name}}/setupbase.py

```diff
@@ -102,36 +128,69 @@ def update_package_data(distribution):
     build_py.finalize_options()
 
 
-def create_cmdclass(wrappers=None):
-    """Create a command class with the given optional wrappers.
+def create_cmdclass(prerelease_cmd=None, package_data_spec=None,
+        data_files_spec=None):
+    """Create a command class with the given optional prerelease class.
 
     Parameters
     ----------
-    wrappers: list(str), optional
-        The cmdclass names to run before running other commands
+    prerelease_cmd: (name, Command) tuple, optional
+        The command to run before releasing.
+    package_data_spec: dict, optional
+        A dictionary whose keys are the dotted package names and
+        whose values are a list of glob patterns.
+    data_files_spec: list, optional
+        A list of (path, patterns) tuples where the path is the
+        `data_files` install path and the patterns are glob patterns.
+
+    Notes
+    -----
+    We use specs so that we can find the files *after* the build
+    command has run.
+    The glob patterns can contain at most one '**' per pattern.
     """
-    egg = bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled
```

Comment:
What's the rationale for removing this? Is it no longer needed for some reason?

Reply:
Or rather I should ask, why are you switching to conditional use of setuptools?

Reply:
I was following the convention in the ipython repo, where setuptools is only invoked when specifically required since it's behavior is not always desirable.